resources/functions/tenant-config/index.py

```diff
@@ -66,12 +94,24 @@ def get_tenant_config_via_req_param(tenant_name):
         logger.error(f"Tenant name not found in path!")
         raise BadRequestError(f"Tenant name not found in path!")
 
-    return _get_tenant_config_for_tenant(tenant_name)
+    return _get_tenant_config_for_tenant_name(tenant_name)
 
 
 @app.get("/tenant-config")
 @tracer.capture_method
-def get_tenant_config_via_header():
+def get_tenant_config_via_param_or_header():
+    tenant_id: str = app.current_event.get_query_string_value(name="tenantId", default_value="")
+    tenant_name: str = ''
+    logger.info(f"tenant_id: {tenant_id}")
+    if tenant_id is None:
+        logger.info(f"No tenantId query parameter found. Looking for tenantName.")
+    else:
+        return _get_tenant_config_for_tenant_id(tenant_id)
+
+    tenant_name = app.current_event.get_query_string_value(name="tenantName", default_value="")
+    if tenant_name is None:
```

Comment:
what happens if `tenant_name` is not `None`?

Reply:
Yeah, good catch. This better?